Susi/Controllers/ChatViewController/SpeechRecognizerDelegate.swift

```diff
@@ -172,8 +172,12 @@ extension ChatViewController: SFSpeechRecognizerDelegate, AVSpeechSynthesizerDel
             let speechUtterance = AVSpeechUtterance(string: message.message)
             speechSynthesizer.delegate = self
 
-            if let language = message.answerData?.language {
-                speechUtterance.voice = AVSpeechSynthesisVoice(language: language)
+            if let selectedLanguage = UserDefaults.standard.object(forKey: ControllerConstants.UserDefaultsKeys.languageCode) as? String {
```

Comment:
Shouldn't this come only from the server?

Reply:
In last meeting, mario told us to implement the feature to change the voice accents from the client side. From server side only one voice language is available.  Refer:https://github.com/fossasia/susi_android/pull/1320